src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -520,6 +520,16 @@ export class IntegrationListDirectory extends AsyncComponent<
                   {tct('No Integrations found for "[searchTerm]".', {
                     searchTerm: searchInput,
                   })}
+                  <p style={{fontWeight: 'bold'}}>
```

Comment:
@ceorourke You should use a styled component to take care of the styling instead of doing it inline like this